io/io/src/RRawFile.cxx

```diff
@@ -29,6 +29,9 @@
 #include <cstring>
 #include <stdexcept>
 #include <string>
+#include <iostream>
+
+#define TESTING_MODE
```

Comment:
Instead, this probably should be set (conditionally) from the compiler command line, depending on the CMake configuration.

If you need help with this, we can discuss offline as required.